src/components/NewsAndArticles/index.jsx

```diff
@@ -98,7 +98,7 @@ const News = () => {
     );
   }
 
-  if (data) {
+  if (!loading && !error && data) {
```

Comment:
Can you please explain why not just data?

Reply:
🤔 May be overthinking.
 I thought that maybe after loading directly some error occurs.
so I wanted to make sure 100% that loading is finished and no errors. 

Or I may be wrong.

Reply:
Over-engineering

Reply:
Yea I think you are overthinking it. it's single threaded so the first 2 conditions will always be checked first. you can leave it if you want i just think it's not necessary